### PR TITLE
[QBot] Docs and Paginated API fix

### PR DESCRIPTION
**Pull request by juna-gupta**
1. Updated docs with the right command to remove AzureRM
2. Fixed API call to fetch members with app token.

### CODE REVIEW

File: samples/msteams-application-qbot/Deployment/README.md [hunk 1]
```diff
@@ -50,6 +50,11 @@ The script will write deployment logs to `deployment.log` file in the `scripts`
 
 Make sure that you have upgraded to Powershell [5.1.22000.653 or later](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2). Run `$PSVersionTable.PSVersion` to check the installed version.
 
+### Max students in a Team
+The solution is tested for upto 1k users in a Team. If your tenant has more than 1k users in a Team, we suggest you to make a few changes to the app logic.
```
Comment (aosolis) [inline samples/msteams-application-qbot/Deployment/README.md:54]: Let's spell out 1,000 and lowercase Team ("up to 1,000 users in a team")

Comment (aosolis) [reply samples/msteams-application-qbot/Deployment/README.md:54]: Also see the capitalized "Team" in the paragraph below

**Review by aosolis — APPROVED** [review]
Approved w/ minor editorial suggestions

File: samples/msteams-application-qbot/Deployment/README.md (hunk 1 above)
Comment (juna-gupta) [reply samples/msteams-application-qbot/Deployment/README.md]: Fixed.

**Review by beauzeaux — APPROVED** [review]

**Pull request merged**